platform/kube/kube_test.go

```diff
@@ -268,9 +268,9 @@ func TestProxyConfig(t *testing.T) {
 
 	key := model.Key{Kind: model.RouteRule, Name: "test", Namespace: ns}
 
-	err := cl.Put(key, rule)
+	err := cl.Post(key, rule)
```

Comment:
(nit) These two lines could be combined, e.g.

if err := cl.Post(key, rule); err != nil {
    t.Errorf(...)
}